tools/schemacode/bidsschematools/validator.py

```diff
@@ -728,6 +794,9 @@ def validate_bids(
     ----------
     paths : str or list of str
```

Comment:
I do like just `paths` better than some `bids_paths` which is in the signature of the function.  Please unify one way or another though.

Reply:
ok